app/src/main/java/org/feature/fox/coffee_counter/ui/authentication/AuthenticationView.kt

```diff
@@ -5,11 +5,8 @@ import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.text.ClickableText
 import androidx.compose.material.Checkbox
 import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
```

Comment:
## General
Thanks for the input!
I've read a bit about and it seems that this seems to be another religion-war in the software world like tabs vs spaces.
[This thread](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad) shows that pretty well. 

### Clean Code Book
The Clean Code Book recommends the usage of wildcards.
> Wildcard imports can sometimes cause name conflicts and ambiguities. Two classes
with the same name, but in different packages, will need to be specifically imported, or at
least specifically qualified when used. This can be a nuisance but is rare enough that using
wildcard imports is still generally better than specific imports.

### IDE
In this case the IDE also organized it with the help of a wildcard and because of that i think it is acceptable to use wildcards. Otherwise it would be the default to avoid wildcards, I guess.

### Performance
Since in the end there is only Bytecode produced it does not affect runtime performance. It does affect compile-time however by a little ([source](https://tharakamd-12.medium.com/is-it-bad-to-use-wildcard-imports-in-java-1b46a863b2be))

### Git
When using a `.*` on 2+ imports where it is more likely that additional imports from that package might follow a wildcard import reduces changes in the version history.

### Solution to import conflicts
If you would like to import a `Composable` problematic package imports could be resolved by the following example:
```kotlin
import morphclue.compose.*
import androidx.compose.*
import androidx.compose.Composable
```

### Note
Of course the Clean Code Book is not and shouldn't be treated as the the bible. Especially, because the authors argument is that it removes clutter when having over 80 imports. This contradicts himself where it should be avoided to use "God Classes". He also states that:
> Long lists of imports are daunting to the reader.

But let us be real. Who reads imports these days? I only have a short look at them when they are highlighted as a warning, error or unused.

#### Conclusion
I'll change it, because for me it makes no difference and I understand your point.
The above points were still interesting to find out and to research so I've wanted to share them with you.

Reply:
Should be resolved in [7d3fdf2](https://github.com/Morphclue/code-camp-2/pull/69/commits/7d3fdf23d81ed0affcba616ba2463e220958c11c).